code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -316,6 +334,11 @@
 		M.adjustBruteLoss(-0.5*REM, 0)
 		M.adjustFireLoss(-0.5*REM, 0)
 		. = TRUE
+	//Fermichem
```

Comment:
add the effects to description so people know it helps stabilize ph

Reply:
also why does it only do it to mobs shouldn't it do it to all reagent containers/always..? or no?

Reply:
??? When did I write this?

Reply:
I think I added this for when I was toying with the idea of making extreme pH in mobs damaging to the mob.

Reply:
I've removed it for now.

Reply:
Though I do like the idea of a neutral pH buffer, I'll add it in Fermichem2